Src/WitsmlExplorer.Frontend/components/ContentViews/EditInterval.tsx

```diff
@@ -43,25 +77,27 @@ const EditInterval = (): React.ReactElement => {
         <StyledLabel label="Start Index" />
         <StyledTextField
           id="startIndex"
-          defaultValue={startIndex}
+          defaultValue={setDefaultValue(startIndex)}
+          variant={isValidStart ? undefined : "error"}
```

Comment:
If you don't use step="1", and you select a date from the picker, there is a bug with the seconds in the input afterwards. Color changes, can't reach it when hitting Tab etc.
![image](https://github.com/equinor/witsml-explorer/assets/70512270/a2339678-6747-47ab-b290-d8269bd25de6)

Reply:
fixed